frontends/precinct-scanner/src/screens/scan_warning_screen.test.tsx

```diff
@@ -158,8 +166,10 @@ test('unreadable', () => {
     ],
   });
 
-  screen.getByRole('heading', { name: 'Review Your Ballot' });
-  userEvent.click(screen.getByRole('button', { name: 'Count my ballot' }));
-  userEvent.click(screen.getByRole('button', { name: 'Yes, count my ballot' }));
+  screen.getByRole('heading', { name: 'Can’t Interpret Ballot' });
```

Comment:
I have never seen this screen. Does it actually occur? We should be going to a scan error screen in this case and not giving the voter an opportunity to cast an uninterpretable ballot.

Reply:
@jonahkagan - Your thoughts here?

Reply:
I don't know how to make it happen either.

Here are the possible adjudication reasons:
```ts
export enum AdjudicationReason {
  UninterpretableBallot = 'UninterpretableBallot',
  MarginalMark = 'MarginalMark',
  Overvote = 'Overvote',
  Undervote = 'Undervote',
  BlankBallot = 'BlankBallot',
}
```

It looks like Overvote, Undervote, and BlankBallot are handled by other screens, so that leaves just UninterpretableBallot and MarginalMark.

As far as I can tell, UninterpretableBallot only occurs [when we can't correctly identify the contests](https://github.com/votingworks/vxsuite/blob/ad22e6c64fce8e4bcbbd17f2656d40a947f1a009/libs/utils/src/hmpb/ballot_adjudication_reasons.ts#L28), but I'm not actually sure why that happens and why we don't just return an `UnreadablePage` or `UninterpretedHmpbPage` interpretation.

MarginalMark I also don't know much about.

Maybe @eventualbuddha or @benadida can shed some light

Reply:
`MarginalMark` is for future use, not currently in use.

`UninterpretableBallot`... I'm not entirely sure! Is it important to determine now? If so I can dig.

Reply:
I think @mattroe should weigh in on how important this is to figure out now.

Reply:
Not important to figure out now. I suggest we present the generic scanning error screen in the case of this improbable uninterpretable ballot case.

Reply:
Uninterpretable Ballot state has been updated to look like this:

<img width="1310" alt="image" src="https://user-images.githubusercontent.com/28444/183525776-ddad237a-e0e0-4983-b95b-66870215c673.png">